src/sqlfluff/core/rules/std/L036.py

```diff
@@ -1,10 +1,22 @@
 """Implementation of Rule L036."""
 
-from collections import namedtuple
+from typing import List, NamedTuple
 
-from sqlfluff.core.rules.base import BaseCrawler, LintResult
+from sqlfluff.core.parser import BaseSegment
+from sqlfluff.core.rules.base import BaseCrawler, LintFix, LintResult
+from sqlfluff.core.rules.doc_decorators import document_fix_compatible
 
 
+class EvalResult(NamedTuple):
+    cnt_select_targets: int
+    select_idx: int
+    first_new_line_idx: int
+    first_select_target_idx: int
+    first_whitespace_idx: int
+    select_targets: List[BaseSegment]
```

Comment:
For now, only the first item in this list is used, but depending on the outcome of the question I raised, the whole list may be useful.